docs/faq/onboarding/lukso-standards.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_label: 'LUKSO Standards'
+sidebar_position: 3
+---
+
+# LUKSO Standard Proposals
+
+### What are LUKSO Standard Proposals?
+
+LUKSO Standard Proposals, often abbreviated as LSPs, serve as building blocks designed to create a wide range of features and possibilities for a new creative economy of blockchain accounts.
+
+With LSPs, the mission is to have standards that facilitate using tools that abstract the blockchain complexity, effectively minimizing the technical problems users may encounter. The goal is to **make blockchain technology as user-friendly and accessible** as the standard Web 2.0 technologies. This ease of use paves the way for broader acceptance and integration of blockchain systems in everyday digital interactions.
+
+### Who can utilize LUKSO Standard Proposals?
+
+LSPs can be seen as a public good and are entirely open source. They can be directly implemented on any EVM blockchain. While LSPs are implemented in Solidity by default, LUKSO ensures that the standardizations are kept platform-independent to be rewritten for other languages or blockchain architecture.
```

Comment:
don't hesitate to make cross references with these pages:

https://docs.lukso.tech/standards/introduction

(with relate links!)